src/Config/maxfactor-support.php

```diff
@@ -8,4 +8,5 @@
         \Symfony\Component\HttpFoundation\Response::HTTP_TEMPORARY_REDIRECT,
         \Symfony\Component\HttpFoundation\Response::HTTP_PERMANENTLY_REDIRECT,
     ],
+    'defaultBreadcrumb' => 'Home',
```

Comment:
Could we please call this `homeBreadcrumb` just to make it really self explanatory?

Reply:
Yeah that makes sense, I'll get that changed.